manifests/sysctl.pp

```diff
@@ -233,25 +235,28 @@
         sysctl { 'kernel.exec-shield': value => $kernel__exec_shield }
       }
 
-      unless $ipv6.is_a(Undef) {
+      if $ipv6.is_a(Undef) {
```

Comment:
I remember the behavior of this section of code being finicky, and I'm not sure adding the ipv6 fact to the spec test keeps the behavior consistent